packages/edit-site/src/store/actions.js

```diff
@@ -373,12 +373,20 @@ export function updateSettings( settings ) {
  * @param {boolean} isOpen If true, opens the list view. If false, closes it.
  *                         It does not toggle the state, but sets it directly.
  */
-export function setIsListViewOpened( isOpen ) {
-	return {
-		type: 'SET_IS_LIST_VIEW_OPENED',
-		isOpen,
+export const setIsListViewOpened =
+	( isOpen ) =>
+	( { dispatch, registry } ) => {
+		const distractionFreeMode = registry
```

Comment:
Since this variable is a boolean value, how about changing it to `isDistractionFree`, as in many other places? The same applies to `distractionFreeMode` included in this PR.